src/services/deleteBlacklistedToken.js

```diff
@@ -0,0 +1,19 @@
+import db from '../database/models';
+
+const { BlacklistedToken } = db;
+
+const deleteExpiredBlacklistedToken = async () => {
+  const tokens = await BlacklistedToken.findAll();
```

Comment:
Instead of getting all the tokens and looping through this list, don't you think it will make sense to just `findAll where current time is >= expiredTime` and then delete while doing something like this `BlacklistedToken.destroy({ where: { id: [1,2,3,4] }})`

Reply:
Alright. I'm on it